test/activityrepo-test.js

```diff
@@ -1,4 +1,4 @@
-import { expect } from 'chai';
+import { expect, AssertionError } from 'chai';
```

Comment:
I may not need this but added it in when testing for failures in project

Reply:
What's it do?

Reply:
You throw an error on purpose and expect an AssertionError